HANDLING_CODE_OF_CONDUCT_REPORTS.md

```diff
@@ -4,69 +4,40 @@ This document describes the proposed process for handling reports and escalation
 This process covers two types of reports based on the [Foundation's Code of Conduct Requirements](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md):
 
 * Reports for spaces managed by the Cross Project Council (CPC) which come in through `report@opensjsf.org`
```

Comment:
In this bullet point, reports@openjsf.org is called out specifically. What about the second bullet point? It seems that address would work as well. In fact, I think the most recent incident used that address and fell into this second bullet point bucket.